packages/clerk-js/src/ui/common/Billing/BillingProvider.tsx

```diff
@@ -17,6 +17,7 @@ type BillingProviderValue = {
   currentPage: BillingPages;
   goToPlanAndBilling: () => void;
   goToManageBillingPlan: () => void;
+  checkPermissions?: boolean;
```

Comment:
I imagine that this is needed to us to support Billing inside the UserProfile where permissions do not exist, right ?

Reply:
Correct @panteliselef